src/field_theory/separable.lean

```diff
@@ -482,3 +508,26 @@ end
 the minimal polynomial of every `x : K` is separable. -/
 @[class] def is_separable (F K : Sort*) [field F] [field K] [algebra F K] : Prop :=
 ∀ x : K, ∃ H : is_integral F x, (minimal_polynomial H).separable
+
+section is_separable_tower
+variables (F K E : Type*) [field F] [field K] [field E] [algebra F K] [algebra F E] [algebra K E]
+  [is_scalar_tower F K E]
+
+lemma is_separable_tower_top_of_is_separable_tower (h : is_separable F E) : is_separable K E :=
+λ x, Exists.cases_on (h x) (λ hx hs, ⟨is_integral_of_is_scalar_tower x hx,
+hs.map.of_dvd (minimal_polynomial.dvd_map_of_is_scalar_tower K hx)⟩)
+
+lemma is_separable_tower_bot_of_is_separable_tower (h : is_separable F E) : is_separable F K :=
```

Comment:
I have to admit, I preferred the old names :) Another option would be the following suggestion, which allows us to write `hFE.tower_top K : is_separable K E` if we have `hFE : is_separable F E`.

```suggestion
lemma is_separable.tower_top (h : is_separable F E) : is_separable K E :=
λ x, Exists.cases_on (h x) (λ hx hs, ⟨is_integral_of_is_scalar_tower x hx,
  hs.map.of_dvd (minimal_polynomial.dvd_map_of_is_scalar_tower K hx)⟩)

lemma is_separable.tower_bot (h : is_separable F E) : is_separable F K :=
```

Reply:
I also liked the old names. I changed the name to this to match `is_integral_tower_top_of_is_integral`. Though now I don't know why I put the last `tower` at the end.

Reply:
I'm happy to change the name back or keep it as `is_separable_tower_top_of_is_separable`, whatever you think is best.

Reply:
Okay, for now I've changed the names to `is_separable_tower_top_of_is_separable` and `is_separable_tower_bot_of_is_separable` in analogy to `is_integral_tower_top_of_is_integral` and `is_integral_tower_bot_of_is_integral`. Let me know if you think I should change them back to the original names (including if you think I should also change the `is_integral` lemma names).

Reply:
Consistency sounds like a good choice, thanks.